### ISSUE TITLE
"lit" command on pyspark doesn't seem to be supported

### ISSUE THREAD

**Issue by souravagasti** [eakmanrq/sqlframe#19]
Getting an error with the following code:

![image](https://github.com/eakmanrq/sqlframe/assets/40858235/53a5f1f9-c4fd-48d9-a9ca-13d0b07d1c31)

**Comment by eakmanrq** [comment]
Lit does work you just need to make sure to import from SQLFrame path instead of PySpark.

```
from sqlframe.standalone import StandaloneSession
from sqlframe.standalone.functions import lit

.....
```

When using SQLFrame you no longer import from pyspark but instead import from sqlframe. I will update docs this evening to make this clearer.

**Comment by eakmanrq** [comment]
PR: https://github.com/eakmanrq/sqlframe/pull/20

**Comment by 1harshit1** [comment]
I see a strange behaviour,
<img width="649" alt="image" src="https://github.com/eakmanrq/sqlframe/assets/49161895/f17d8729-d50b-4112-bfe6-ed89272dd291">

The code works fine with line 21, but does not work with line 20. (It is the same step in different order).
<img width="654" alt="image" src="https://github.com/eakmanrq/sqlframe/assets/49161895/70261408-a4ed-4541-a2db-2ef7e9e81968">

**Comment by eakmanrq** [comment]
I think if you replace `df.Name + df.Age` to `col("name") + col("age")` then it will work. Will look into fix tonight.

**Comment by eakmanrq** [comment]
@1harshit1 This should be fixed in `1.1.3`. If you have further issues please open a new issue and I would be happy to take a look. Thanks for reporting this!
